lib/index.coffee

```diff
@@ -1,6 +1,7 @@
 {parse} = require './parser'
 cheerio = require 'cheerio'
 clone = require 'lodash.clone'
+jsesc = require 'jsesc'
```

Comment:
I did some local tests with `jsesc` and it worked out quite well, but does not *exactly* resemble the encoding Tumblr does (e.g., `<` is not encoded to `\x3c`, but it still works).